dgassist/README.md

```diff
@@ -0,0 +1,5 @@
+# /sri/
```

Comment:
Typo.

Reply:
```suggestion
# /dgassist/
```

Reply:
You are right, thanks for point this out. I'll fix this in the next PR, there might be some changes next week anyway.